README.md

```diff
@@ -41,6 +48,7 @@ docker run -v ./data:/data --env WANDB_API_KEY=<WANDB_API_KEY> deep-learning-pro
 # evaluation
 docker run -v ./data:/data --env WANDB_API_KEY=<WANDB_API_KEY> deep-learning-project --pipeline evaluation
 ```
+TODO: add inference pipline run command
```

Comment:
I thought that the evaluation pipeline was the inference pipeline

Reply:
I haven't noticed it 😅 
I had a similar intuition as you had that we can make a kedro run call for the API.

I'll remove it.
```suggestion
```